os/Makefile.unix

```diff
@@ -476,7 +476,8 @@ ifeq ($(CONFIG_RAW_BINARY),y)
 	$(Q) $(OBJCOPY) $(OBJCOPYARGS) -O binary $(BIN) $(BIN).bin
 endif
 
-# Verify app binary to make sure there are no undefined symbols
+# Verify app binary to make sure there are no undefined symbols and to check the l2 pages per app are sufficient enough
+ifeq ($(CONFIG_ARCH_HAVE_MMU),y)
```

Comment:
Instead of adding condition here, you can add condition just for calling the mkverifyappsize.py.
Also, you need to check for CONFIG_ARCH_USE_MMU... not HAVE_MMU.